darts/models/forecasting/nlinear.py

```diff
@@ -147,7 +147,7 @@ def forward(
             )
 
             if self.normalize:
-                x = x + seq_last  # Note: works only when nr_params == 1
+                x = x + seq_last[:, :, :x.shape[-1]] # match last dimension
```

Comment:
This looks like a smart solution. Idea how to make it more explicit:

```suggestion
                # make sure to only copy nr_params * number_of_targets
                x = x + seq_last[:, :, :x.shape[-1]]
```